fineract-provider/src/main/java/org/apache/fineract/portfolio/loanaccount/domain/LoanAccountDomainServiceJpa.java

```diff
@@ -405,6 +408,8 @@ public LoanTransaction makeChargePayment(final Loan loan, final Long chargeId, f
         }
 
         postJournalEntries(loan, existingTransactionIds, existingReversedTransactionIds, isAccountTransfer);
+        loanAccrualTransactionBusinessEventService.raiseBusinessEventForAccrualTransactions(loan, existingTransactionIds);
```

Comment:
Please raise the event after the "recalculateAccruals" method which might created new Accrual transaction as well.